### ISSUE TITLE
Can't get past that first Bowser battle; stuck in a loop.

### ISSUE THREAD

**Issue by Daliyasincsgxds** [Brotenko/PMMasterQuest#21]
The game shows up a retry menu at the end of the first battle--the one where you're supposed to lose.

**Comment by gamezoid123** [comment]
Yeah, just figured it out. They changed the prologue. You have to talk to the toad just before the stair way into peach's room.
